app/models/api/transliterate.php

```diff
@@ -0,0 +1,6 @@
+<?php
+
+$t = Transliterator::create('Serbian-Latin/BGN');
+$translitareted_string = $t->transliterate($request->parameters[2]);
+
+return $json = ['translitareted_string' => $translitareted_string];
```

Comment:
Typo in the variable name. 

Also I don't think the extension is enabled by default (but didn't have a chance to test this yet). If that's the case, you should check if the class is available before trying to use it.